darwin/datatypes.py

```diff
@@ -34,7 +34,11 @@
 ErrorHandler = Callable[[int, str], None]
 
 ItemId = Union[str, int]
-JSONFreeForm = Dict[str, Any]  # type: ignore
+
+# Types that assist in handling JSON payloads
+JSONFreeForm = Dict[str, UnknownType]
```

Comment:
I simplified these whilst working in the area as they were causing issues with type hints anyway, and didn't need to be as complex.